src/v2/Apps/Home/Components/HomeHeroUnits/HomeHeroUnit.tsx

```diff
@@ -203,19 +204,30 @@ export const HomeHeroUnit: React.FC<HomeHeroUnitProps> = ({
   )
 
   return (
-    <GridColumns bg={bg} width="100%">
-      {layout === "a" ? (
-        <>
-          {figure}
-          {description}
-        </>
-      ) : (
-        <>
-          {description}
-          {figure}
-        </>
+    <>
+      {image && index === 0 && (
+        <Link
+          rel="preload"
+          as="image"
+          href={image.src}
+          imagesrcset={image.srcSet}
```

Comment:
How one goes about preloading a `srcset`.  The browser should preload the correct variant.